bin/build-doc

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
```

Comment:
Could you please move this file to `scripts/`? This is the folder that contains the non-production scripts in this repo. Also, I'd rename the script to something like `update-dispensary-doc`.